src/main/java/com/example/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Table(name = "cart_items")
+@Entity
+public class CartItem {
```

Comment:
Maybe add equals and hashCode

Reply:
agree, let's add it and toString, but exclude lazy fields from all of that